README.md

```diff
@@ -34,3 +34,48 @@ There are numerous optional inputs that can be used to configure the action:
 - `paths`: A comma separated list of paths to scan. If not provided, all paths in the repository will be scanned.
 - `policy_ids`: A comma separated list of policy IDs to execute. If not provided, all policies in the KSOC account will be executed.
 - `policy_tags`: A comma separated list of policy tags to execute. If not provided, all policies in the KSOC account will be executed.
+
+KSOC Guard Action is storing the results of the scan in the output called `results`. This can be used to create a comment on the PR with the results of the scan. The following example shows how to do this (note that `pull-resuests` permission is required for this):
+
+```yaml
+name: ksoc-guard
+
+on:
+  pull_request:
+
+jobs:
+  ksoc-guard:
+    permissions:
+      contents: read
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: KSOC Guard
+        uses: ksoclabs/guard-action@main
+        with:
+          ksoc_account_id: <KSOC_ACCOUNT_ID>
+          ksoc_access_key_id: ${{ secrets.KSOC_ACCESS_KEY_ID }}
+          ksoc_secret_key: ${{ secrets.KSOC_SECRET_KEY }}
+      - name: comment
+        if: success() || failure()
+        uses: actions/github-script@v6
+        env:
+          STATUS: ${{ steps.ksoc-guard.outcome == 'success' && 'success ✅' || 'failure ❌' }}
+        with:
+          script: |
+            const output = `
+              Policy Executor: ${{ env.STATUS }}
+
+              \`\`\`
+              ${{ steps.ksoc-guard.outputs.results }}
+              \`\`\`
+              `
+            github.rest.issues.createComment({
```

Comment:
Couldn't our GH action do it? With a config option?

Reply:
No, GH action is a single step of a workflow, it should be as small as possible and do one thing. Not everyone wants to use PR comments in their workflows.

Reply:
Makes sense. I guess we can add this next step.